internal/incident/incidents.go

```diff
@@ -19,6 +20,89 @@ var (
 	currentIncidentsMu sync.Mutex
 )
 
+// LoadOpenIncidents loads all active (not yet closed) incidents from the database and restores all their states.
+// Returns error ony database failure.
+func LoadOpenIncidents(
```

Comment:
But in general, this function looks pretty similar to `GetCurrent()` and duplicates it quite a bit.

Shouldn't it be possible to simplify this function to something like this pseudocode:
```
for id in `select object_id from incident where recovered_at is null` {
  obj = LoadObjFromDB(id)
  incident = GetCurrentIncident(obj)
  incident.RestoreTimers()
}
```

Yes, that queries the `incident` table twice, but I'd rather keep the code simpler for now, as there'll be some quite intensive schema changes soon and we can optimize something like this after that was done.